src/System.Diagnostics.DiagnosticSource/tests/HttpHandlerDiagnosticListenerTests.cs

```diff
@@ -421,6 +421,37 @@ public async Task TestActivityIsCreated()
             parentActivity.Stop();
         }
 
+
+        [OuterLoop]
+        [Fact]
+        public async Task TestInvalidBaggage()
+        {
+            var parentActivity = new Activity("parent")
+                .AddBaggage("key", "value")
+                .AddBaggage("bad/key", "value")
+                .AddBaggage("goodkey", "bad/value")
+                .Start();
+            using (var eventRecords = new EventObserverAndRecorder())
+            {
+                using (var client = new HttpClient())
+                {
+                    (await client.GetAsync(Configuration.Http.RemoteEchoServer)).Dispose();
+                }
+
+                Assert.Equal(1, eventRecords.Records.Count(rec => rec.Key.EndsWith("Start")));
+                Assert.Equal(1, eventRecords.Records.Count(rec => rec.Key.EndsWith("Stop")));
+
+                WebRequest thisRequest = ReadPublicProperty<WebRequest>(eventRecords.Records.First().Value, "Request");
+                var correlationContext = thisRequest.Headers["Correlation-Context"].Split(',');
+
+                Assert.Equal(3, correlationContext.Count());
```

Comment:
Assuming correlationContext is a  string[], this should be .Length rather than .Count()

Reply:
@terrajobst, here's another example about why var makes code reviews harder... it's not easy for me to determine what the type of correlationContext is, so I don't know whether Count() is the most appropriate thing to use here. cc: @ahsonkhan.  This is just a trivial example... there are lots in more impactful places that show up all the time when var is used and the type isn't obvious.

Reply:
Agreed :)